testchain/runtime/mainnet/src/lib.rs

```diff
@@ -623,15 +621,12 @@ impl pooled_rewards::Config for Runtime {
 }
 
 construct_runtime! {
-	pub enum Runtime where
-		Block = Block,
-		NodeBlock = primitives::Block,
-		UncheckedExtrinsic = UncheckedExtrinsic
+	pub enum Runtime 
 	{
-		System: frame_system::{Pallet, Call, Storage, Config, Event<T>} = 0,
+		System: frame_system::{Pallet, Call, Storage, Config<T>, Event<T>} = 0,
```

Comment:
We could then also remove the explicit parameters (e.g., `{Pallet, Call, Storage, Config<T>, Event<T>}`) in this `construct_runtime!`.

Reply:
Let's do that!